src/js/jsx/sections/layers/LayerFace.jsx

```diff
@@ -617,10 +591,29 @@ define(function (require, exports, module) {
                 }
             }
 
+            var layerList;
+
+            if (hasChildren && (layer.expanded || this._isExpanded)) {
+                this._isExpanded = true;
+                
+                var LayersList = require("jsx!./LayersList");
```

Comment:
`LayerList` is also requiring `LayerFace`, so we need to put the require statement in the function scope, otherwise `LayersList` will be an empty object.